content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -26,6 +25,42 @@ Amazon EC2's `m3.large` VM size may be appropriate depending on your workload. A
 **Tip:** For recommendations beyond the minimum requirements, see [Capacity Planning](capacity-planning.html).
 
 
+## Installation on Ubuntu with alien
```

Comment:
We state that ubuntu has received the highest level of QA and testing - so I moved it above centos and macOS install sections. Okay?